pkgs/applications/misc/keystore-explorer/default.nix

```
@@ -0,0 +1,28 @@
+{ fetchzip, stdenv, jre }:
```

jre is almost no longer a thing:

```suggestion
{ fetchzip, stdenv, jdk }:
```

Ever since: https://github.com/NixOS/nixpkgs/pull/89731